models/servicenow__user_aggregated.sql

```diff
@@ -1,4 +1,4 @@
-{{ config(enabled=var('servicenow__using_roles', False)) }}
+{{ config(enabled=var('servicenow__using_roles', True)) }}
```

Comment:
Can you also add the same enabled config to the relevant validation tests ([example](https://github.com/fivetran/dbt_servicenow/blob/2edd407a2c1130e460d3eb881776268aff08d716/integration_tests/tests/consistency/user_aggregated_aggs.sql#L3)). This way when running validation tests we avoid false failures if the models are disabled when testing between dev and prod.

Reply:
Added.